src/modules/auth/auth.controller.ts

```diff
@@ -58,4 +60,22 @@ export default class RegistrationController {
     const { id: user_id } = request['user'];
     return this.authService.enable2FA(user_id, password);
   }
+
+  @skipAuth()
+  @Post('signin-token')
+  @ApiOperation({ summary: 'Request Signin Token' })
+  @ApiResponse({ status: 200, description: 'Sign-in token sent to email', type: RequestSigninTokenDto })
+  @ApiResponse({ status: 400, description: 'Bad request' })
```

Comment:
Describe Validation error

```suggestion
@ApiResponse({ status: 400, description: 'Bad request' })
@ApiResponse({ status: 401, description: 'Unauthorized request' })
```